edx_django_utils/monitoring/utils.py

```diff
@@ -4,7 +4,7 @@
 
 Usage:
 
-    from openedx.core.djangoapps import monitoring_utils
+    from edx_django_utils import monitoring as monitoring_utils
```

Comment:
Should this be `from edx_django_utils.monitoring import utils as monitoring_utils`?

Reply:
The public interface is in `__init__.py`.